src/ubuntu/build-scripts/build-rootfs.sh

```diff
@@ -32,7 +32,7 @@ do
 
     echo "Using $dockerCrossDepsTag to clone arcade to fetch scripts used to build cross-toolset"
     docker exec $buildRootFSContainer \
-        git clone https://github.com/dotnet/arcade /scripts
+        git clone --depth 1 https://github.com/dotnet/arcade /scripts
```

Comment:
👍 also `--single-branch` helps, since all the rest of the non-master branches (in this case) will be just throw away and wasteful.